modules/core/src/main/java/org/apache/ignite/internal/processors/performancestatistics/FilePerformanceStatisticsWriter.java

```diff
@@ -319,8 +330,41 @@ private void doWrite(OperationType op, int recSize, Consumer<ByteBuffer> writer)
         }
     }
 
+    /** @return {@code True} if string was cached and can be written as hashcode. */
+    private boolean stringCached(String str) {
```

Comment:
Let's rename this `cacheIfPossible`.

Reply:
Let's move this method to the class end, because utility methods should be in the end of the class.

Reply:
Renamed and moved.